qbft/README.md

```diff
@@ -4,6 +4,8 @@
 ## Introduction
 This is a spec implementation for the QBFT protocol, following [formal verification spec](https://entethalliance.github.io/client-spec/qbft_spec.html#dfn-qbftspecification) / [github repo](https://github.com/ConsenSys/qbft-formal-spec-and-verification).
 
+To understand the implementation and its comparison to the H. Moniz [IBFT paper](https://arxiv.org/pdf/2002.03613.pdf), please follow this [documentation](docs/README.md).
+
 ## Important note on message processing
 The spec only deals with message process logic but it's also very important the way controller.ProcessMsg is called.
 Message queueing and retry are important as there is no guarantee as to when a message is delivered.
```

Comment:
Yes, I also believe it means reprocessing since it regards the controller.ProcessMsg method and, thus, the message was already received.

Ok. I'll change to "received".